File: 3. Semestar/Napredno programiranje i programski jezici/Vezbe/Vezbe 3/1/pravougaonik.cpp
#include "pravougaonik.hpp"

Pravougaonik::Pravougaonik() {
    a = 1;
    b = 1;
}

Pravougaonik::Pravougaonik(double a, double b) {
    this->a = a;
    this->b = b;
}

double Pravougaonik::getA() {
    return a;
}

double Pravougaonik::getB() {
    return b;
}

double Pravougaonik::getO() {
    return 2 * (a + b);
}

double Pravougaonik::getP() {
    return a * b;
}

File: 3. Semestar/Napredno programiranje i programski jezici/Vezbe - Nasledjeno/vezbe9/Njihova resenja/3/Pravougaonik.java
package z3;

public class Pravougaonik {
	private double a;
	private double b;
	
	public Pravougaonik(double a, double b) {
		this.a = a;
		this.b = b;
	}
	
	public Pravougaonik(Pravougaonik p) {
		this.a = p.a;
		this.b = p.b;
	}
	
	public double getA() {
		return a;
	}
	
	public void setA(double a) {
		this.a = a;
	}
	
	public double getB() {
		return b;
	}
	
	public void setB(double b) {
		this.b = b;
	}
	
	public double getO() {
		return 2 * a + 2 * b;
	}
	
	public double getP() {
		return a * b;
	}

}
